Formula/gst-python.rb

```diff
@@ -27,12 +27,6 @@ class GstPython < Formula
   depends_on "pygobject3"
   depends_on "python@3.11"
 
-  # See https://gitlab.freedesktop.org/gstreamer/gst-python/-/merge_requests/41
-  patch do
-    url "https://gitlab.freedesktop.org/gstreamer/gst-python/-/commit/3e752ede7ed6261681ef3831bc3dbb594f189e76.diff"
-    sha256 "d6522bb29f1894d3d426ee6c262a18669b0759bd084a6d2a2ea1ba0612a80068"
-  end
```

Comment:
This patch needs to be updated not removed. Someone may want to take over upstream MR and submit to monorepo.